README.md

```diff
@@ -1,3 +1,21 @@
-# quizy-smart-dating
+# Quizy Smart Dating
 
-Welcome to quizy smart dating! :)
\ No newline at end of file
+Welcome to Quizy Smart Dating! :)
+
```

Comment:
Let's add a cool demo video and some screenshots I have a lot of ideas how to make it cool here ❤️❤️❤️